sdk/cosmosdb/cosmos/src/retry/retryUtility.ts

```diff
@@ -96,6 +106,11 @@ export async function execute({
       err.substatus === SubStatusCodes.ReadSessionNotAvailable
     ) {
       retryPolicy = retryPolicies.sessionReadRetryPolicy;
+    } else if (
+      err.code === StatusCodes.ServiceUnavailable ||
+      err.code === StatusCodes.RequestTimeout
```

Comment:
In other SDKs timeouts are not related to 408s returned by the service but network timeouts. Looking at the related issue, it is linked at detecting HTTP timeouts. How does the JS SDK detect Http timeouts? Does it surface them as 408 status codes?

Reply:
I understand, I was somewhat perplexed by the Python SDK pull request found at this link: https://github.com/Azure/azure-sdk-for-python/pull/28420. It appears that they were addressing 408 errors, but I agree that it would be more appropriate to update it for 503 errors or TimeoutErrors.

Reply:
I'm not fluent on Python, maybe in that language, Http timeouts are converted to 408s